libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,63 @@
+import { DateRangeDto } from './date-range.dto';
+
+/** Anime data dto. */
+export interface AnimeDto {
+
+	/** Anime id. */
+	readonly id: number;
+
+	/**
+	 * Anime creation date.
+	 * @example 2023-07-18T10:32:17.881Z.
+	 */
+	readonly created: string;
+
+	/**
+	 * Anime modification date.
+	 * @example 2023-07-18T10:32:17.881Z
+	 */
+	readonly modified: string;
+
+	/** English anime title. */
+	readonly title_eng: string;
+
+	/** Japanese anime title. */
+	readonly title_jpn: string;
```

Comment:
Since they are placed in relative DTO we can omit reference to _anime_
```suggestion
	/** English title. */
	readonly title_eng: string;

	/** Japanese title. */
	readonly title_jpn: string;
```